### ISSUE TITLE
Error: open main.tf: no such file or directory (use with pre-commit)

### ISSUE THREAD

**Issue by gui-don** [segmentio/terraform-docs#232]
Hi!

### Prerequisites

- [x] This issue describes a bug.

### Description

Since version 0.9.0, terraform-docs seems to require a `main.tf`. In my company standards, some of our projects does not have a `maint.tf`… 

#### Actual Behavior

When a project does
```
$ pre-commit run -a

Error: open main.tf: no such file or directory
```

#### Expected Behavior

Not sure the problem come from terraform-docs because it still runs as is, however only terraform-docs has changed. Within pre-commit: v0.8.2 works, v0.9.0 does not.
Maybe you could give me useful information.

### Version

erraform-docs version v0.9.0 89b129b linux/amd64 BuildDate: 2020-03-31T20:19:01+0000

**Comment by zopanix** [comment]
@gui-don:
Change you're pre-commit config to this:

```
 repos:
 - repo: git://github.com/antonbabenko/pre-commit-terraform
   rev: v1.27.0
   hooks:
     - id: terraform_fmt
     - id: terraform_docs
+      args: ['-a', '--no-header']
```
But since behavior changed, it should have been a breaking release.

**Comment by gui-don** [comment]
Thanks @zopanix.

**Comment by tbugfinder** [comment]
@zopanix 
As discussed in #152 semver will start with 1.0.0 until then breaking changes could be included.

**Comment by khos2ow** [comment]
@gui-don thanks for reporting this, this was - unfortunately - a regression added in #217 and I've fixed it in #235 , for the time being you can use @zopanix's workaround until `v0.9.1` is released.

**Comment by khos2ow** [comment]
I also completely agree with @tbugfinder, and if we hadn't introduced the regression, `v0.9.0` release wouldn't have been a breaking release anyway. But generally there's no hard promise of backward compatibility for v0.x, although we try our best to do it anyway.

**Comment by khos2ow** [comment]
Fixed by #235
